internal/lib/lib.go

```diff
@@ -541,13 +541,26 @@ func GetSniPoolName(ingName, namespace, host, path, infrasetting string, dedicat
 	return poolName
 }
 
+func GetEncodedSniPGPoolNameforRegex(poolName string) string {
+	hash := sha1.Sum([]byte(poolName))
+	encodedStr := GetNamePrefix() + hex.EncodeToString(hash[:])
+	return encodedStr
+}
+
+func GetEncodedStringGroupName(host, path string) string {
+	hash := sha1.Sum([]byte(host + path))
+	encodedStr := GetAKOUser() + "-" + hex.EncodeToString(hash[:])
+	return encodedStr
+}
+
 func GetSniHttpPolName(namespace, host, infrasetting string) string {
 
 	if infrasetting != "" {
 		return Encode(NamePrefix+infrasetting+"-"+namespace+"-"+host, HTTPPS)
 	}
 	return Encode(NamePrefix+namespace+"-"+host, HTTPPS)
 }
+
```

Comment:
Additional space

Reply:
Intentionally added extra blank line between functions like we follow at most places.